src/pages/themes/author/[id]/[...page].astro

```diff
@@ -46,9 +46,17 @@ if (allPages.length && !page) {
 }
 
 const themes = page.data;
+
+const author = themes[0].Author.name || 'Unknown Author';
+const title = `Themes by ${author}`;
+const description =
+	'Explore a variety of Astro themes — designed to help you launch quickly, with flexible layouts, practical features, and customization built in.';
```

Comment:
Would be nice to have a unique description for each, so maybe we work in the theme author here too something like this?

```suggestion
	'Explore Astro themes by ${author}. Find a design to launch your next site quickly, with flexible layouts, practical features, and more.';
```